workers/unity/spatialos.UnityGameLogic.worker.json

```diff
@@ -52,6 +52,7 @@
       "macos": {
         "command": "open",
         "arguments": [
+          "-n",
           "./build/worker/UnityGameLogic@Mac/UnityGameLogic@Mac.app",
```

Comment:
Does this not need to change to be similar to the managed launch? What happens if the user tries to launch two instances of the game logic externally?

Reply:
nope, the difference is that with external you try to start a worker and with managed the runtime tries to start a worker. therefore, open won't work for managed launch, but works fine for external